packratt/cache.py

```diff
@@ -127,6 +130,9 @@ def __getitem__(self, key):
 
         return entry
 
+    def get_cache_entry(self, key):
```

Comment:
Do we need this? Can we not just use?


```python
value = cache[key]
```

which invokes ``Cache.__getitem__``?